UNUMColorPicker/Classes/UNUMColorPickerViewController.swift

```diff
@@ -13,72 +13,82 @@ public protocol UNUMColorPickerDelegate: class {
     func cancel()
 }
 
-public class UNUMColorPickerView: UIView {
+public class UNUMColorPickerViewController: UIViewController {
```

Comment:
I changed this to a ViewController because it really was a viewController -- even though it was only being considered a UIView. I'll clarify.

A view (in every architecture I can think of: MVC, MVVM, VIPER, etc) is the layout of elements and that sort of logic. In MVC, the xib could be considered the view.

The viewController is the class responsible for all the logic -- I'm being simplistic here, so let me put it another way. Logic should not be contained in the View.

Looking at this class and considering "what here is view logic" and "what here is viewController logic", everything here is viewController logic. It's not even a mixture of view and viewController stuff. So really this should just be made into a viewController.

This still doesn't get us to proper MVVM. There is stuff here that would go into the ViewModel, but I just want to get this library functioning as we'd like, and then we can refactor further later.

Changing this to a ViewController gets us to the desired API and sets us up well for refactoring.